### PR TITLE
storybook: add ClipboardButton story

### PR DESCRIPTION
**Pull request by scarlettperry**
### Description
PR adds the ClipboardButton story.

cbutton

### Testing Performed
Locally

### TODOs
- [x] Add gif

### CODE REVIEW

**Comment by scarlettperry** [comment]
Initially I specified the onClick arg in the story, but when I did, this logic didn't happen: https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/button.tsx#L87. I think the onClick arg may not be needed in the story since there isn't additional logic to trigger outside of what the component already handles?

File: frontend/packages/core/src/stories/clipboard-button.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import type { Meta } from "@storybook/react";
+
+import type { ClipboardButtonProps } from "../button";
+import { ClipboardButton } from "../button";
+
+export default {
+  title: "Core/Clipboard Button",
```
Comment (dschaller) [inline frontend/packages/core/src/stories/clipboard-button.stories.tsx:8]: what do you think of
```suggestion
  title: "Core/Buttons/Clipboard",
```

Comment (scarlettperry) [reply frontend/packages/core/src/stories/clipboard-button.stories.tsx:8]: in this PR, going to move the other button stories under `Core/Buttons/` as well

Comment (scarlettperry) [reply frontend/packages/core/src/stories/clipboard-button.stories.tsx]: Screen Shot 2020-11-04 at 4 53 51 PM

**Review by dschaller — APPROVED** [review]

**Pull request merged**